dxr/plugins/python/analysis.py

```diff
@@ -202,4 +203,70 @@ def analyze_import(self, node):
                     if package_path:
                         import_name = package_path + '.' + import_name
 
-            self.tree_analysis.names[absolute_local_name] = import_name
+            import_tuple = Import(absolute_name=absolute_local_name,
+                                  line=node.lineno, col=node.col_offset,
+                                  import_name=import_name)
+            self.tree_analysis.definition_tree.add(import_tuple)
+
+
+# Namedtuples that are stored in DefinitionTree.
+Definition = namedtuple('Definition', ('absolute_name', 'line', 'col', 'path'))
+Import = namedtuple('Import', ('absolute_name', 'line', 'col', 'import_name'))
```

Comment:
I assume you went with namedtuples for size? I believe attr-based lookup in them is a little slower than a class. Probably a `__slots__`-based class is the best of both worlds; I doubt you need index-based access.